components/suggestion/switch_channel_provider.jsx

```diff
@@ -66,6 +68,14 @@ class SwitchChannelSuggestion extends Suggestion {
 
         const member = this.props.channelMember;
         let badge = null;
+
+        const state = getState();
+        const currentUserId = getCurrentUserId(state);
+        const status = getStatusForUserId(state, currentUserId);
+        const user = getUser(state, currentUserId);
+        const username = user.username;
+        const fullname = `${user.last_name} ${user.first_name}`;
```

Comment:
this should be `firstname lastname` since that's the convention we've been using in other places.